docs/UserGuide.md

```diff
@@ -252,15 +260,22 @@ Sample Output:
 ---
 ### Searching for an applicant : `search`
 
-Allows you to search for applicants containing specified keywords, these keywords could be in **any** of the 
-applicant's name, job, round or skills.
+Allows you to search for applicants containing the input keywords and the keyword has to be an exact word. You can search for the applicant based on 
+name, job, round or skills. 
 
 <br>
 
 Format:
 ```
 search [n/NAME]…​ [j/JOB]…​ [r/ROUND]…​ [s/SKILL]…​
 ```
+* You must provide a full match of keyword you want to search for.<br>
+  e.g. Searching `n/Hans` will match an applicant with name `Hans Lee` but will not match an applicant with name `Han`. <br>
+* The search command is case-insensitive. <br>
+  e.g Searching `n/hans` will match an applicant with name `Hans`.
+* You can search for a combination of keywords. 
+  After a search command, applicants will be displayed in descending order of matched keywords, with the most matched applicant on the top of the list.
```

Comment:
Line 277: Consider changing to .... descending order of (number of) matched keywords, ~~with the most matched applicant on the top of the list.~~ with the applicant with the most number matches at the top of the list